lib/paper_trail/has_paper_trail.rb

```diff
@@ -46,6 +47,12 @@ module ClassMethods
       #   the instance was reified from. Default is `:version`.
       # - :save_changes - Whether or not to save changes to the object_changes
       #   column if it exists. Default is true
+      # - :join_tables - If the model has a has_and_belongs_to_many relation
+      #   with an unpapertrailed model, passing the name of the association to
+      #   the join_tables option will paper trail the join table but not save
+      #   the other model, allowing reification of the association but with the
+      #   other models latest state (if the other model is paper trailed, this
+      #   option does nothing)
```

Comment:
> (if the other model is paper trailed, this option does nothing)

Are you sure that's the case? I just read through the code and don't see where that would happen. It seems like if one side of the association has `join_tables` set, and the other side of the association has paper_trail enabled without `join_tables`, this code will correctly load the versions for both sides of the association.

Perhaps you meant to say that there's no point enabling `join_tables` on both models?